apps/antalmanac/src/components/RightPane/AddedCourses/AddedCoursePane.tsx

```diff
@@ -104,48 +102,26 @@ function ClearScheduleButton() {
     );
 }
 
-function CopyScheduleButton() {
-    const [scheduleNames, setScheduleNames] = useState(AppStore.getScheduleNames());
+function CopyScheduleButton(props: { index: number }) {
```

Comment:
```suggestion
function CopyScheduleButton(index: number) {
```
No need to use a props object for one prop